config/config.go

```diff
@@ -157,6 +157,12 @@ type ServerConfig struct {
 	LogSyslogServerKeyContents      string `ini:"db_log_syslog_server_key_contents"`
 	LogSyslogServerClientCAContents string `ini:"db_log_syslog_server_client_ca_contents"`
 
+	// Configures the collector to start a built-in OTLP HTTP server that
+	// receives logs and optionally filters them by Kubernetes pod or labels
+	LogOtelServer    string `ini:"db_log_otel_server"`
+	LogOtelK8SPod    string `ini:"db_log_otel_k8s_pod"`
+	LogOtelK8SLabels string `ini:"db_log_otel_k8s_labels"`
```

Comment:
It would be nice to have a way to specify this via config vars too, like adding the following in read.go:

```golang
	if logOtelServer := os.Getenv("LOG_OTEL_SERVER"); logOtelServer != "" {
		config.LogOtelServer = logOtelServer
	}
	if logOtelK8SPod := os.Getenv("LOG_OTEL_K8S_POD"); logOtelK8SPod != "" {
		config.LogOtelK8SPod = logOtelK8SPod
	}
	if logOtelK8SLabels := os.Getenv("LOG_OTEL_K8S_LABELS"); logOtelK8SLabels != "" {
		config.LogOtelK8SLabels = logOtelK8SLabels
	}
```